apis/projectcontour/v1/httpproxy.go

```diff
@@ -322,7 +322,7 @@ type HTTPProxy struct {
 
 	Spec HTTPProxySpec `json:"spec"`
 	// +optional
-	Status `json:"status"`
+	Status `json:"status,omitempty"`
```

Comment:
Does this suffer from the same problem as https://github.com/projectcontour/ir2proxy/issues/8; this is a struct field marker `omitempty` -- or is the problem in https://github.com/projectcontour/ir2proxy/issues/8 some compounding factor?

Reply:
ah, 🤦‍♂ 
Yes, you're totally correct. This will need to change to
```go
Status *Status `json:"status,omitempty"`
```
Which is a much bigger change. I'll work on this today.